### DISCUSSION TITLE
How do I disable dynamic menu/state

### DISCUSSION THREAD

**Discussion by gwyndall** [fomantic/Fomantic-UI#1676] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
# Help Wanted
I'm building a static site with Fomantic-UI. When I created my menus, but before I wired them up to links, I noticed that the text of the dropdown link item changed when one of the submenu items was selected. Neat! But since I'm not using anything that preserves state, that change is not sustained, now that the links take you to a new page. I'm fine with that, but I don't like how the text changes briefly before the browser loads the new page. I would like to eliminate that change altogether.

Or, if there's a way to maintain that change on the new page, I'd be fine with that too.

## Testcase
https://jsfiddle.net/gwyndall/b98rd1ew/12/

**Comment by TheJltres** [comment]
If i understand correctly, that can be solved adding an input and a value to that input:  https://jsfiddle.net/TheJltres/0po1stgy/1/

PS: I tried with Behavior 'set selected' but i couldn't get it to work

**Comment by ko2in** [comment, marked as answer]
@gwyndall If you don't want to update the current text when you click the menu item. You can use `action` property with `select` in dropdown [settings](https://fomantic-ui.com/modules/dropdown.html#/settings).
```
$('.ui.dropdown').dropdown({
  action: 'select'
});
```
It won't change the text, and you'll reach to the URL that you specify for your menu item. See fiddle (https://jsfiddle.net/ko2in/htLbewcg/5/).

Since, you're creating the static site, then you can just set the appropriate text for your dropdown menu in that HTML page. Example in the HTML page for 'Item 1' link:
```
<div class="ui dropdown link item">
  <span class="text" role="group">Item 1</span>
  <i class="dropdown icon"></i>
  <div class="menu submenu">
    <div class="header">Services</div>
    <a class="item" href="https://www.google.com">Item 1</a>
    <a class="item" href="https://www.facebook.com">Item 2</a>
    <a class="item" href="https://www.twitter.com">Item 3</a>
  </div>
</div>
```
Is that what you're looking for?

**Comment by lubber-de** [comment]
If the dropdown entries should link to a new webpage immediatly, as @ko2in example, you could even use `ui simple dropdown` which does not need any javascript.
https://jsfiddle.net/zgmw7df5/

**Comment by gwyndall** [comment]
Thank you!  @ko2in, that's exactly what I needed.
